IPIP/0000-gateway-tar-response-format.md

```diff
@@ -0,0 +1,63 @@
+# IPIP 0000: Gateway TAR Response Format
+
+- Start Date: (format: 2022-10-10)
+- Related Issues:
+  - [ipfs/specs/pull/288](https://github.com/ipfs/specs/pull/288)
+  - [ipfs/go-ipfs/pull/9029](https://github.com/ipfs/go-ipfs/pull/9029)
+  - [ipfs/go-ipfs/pull/9034](https://github.com/ipfs/go-ipfs/pull/9034)
+
+## Summary
+
+Add TAR as a response format for the HTTP Gateway.
```

Comment:
Do we have a formal definition of HTTP Gateway (or IPIP)? Should we link to that?